src/button/button-group.scss

```diff
@@ -10,9 +10,7 @@
   align-items: center;
   isolation: isolate;
 
-  @if $orientation == 'horizontal' {
-    max-width: 100%;
-  } @else {
+  @if $orientation != 'horizontal' {
```

Comment:
so just `== 'vertical'`

Reply:
This is the personal convention I chose for this component. See https://github.com/iTwin/iTwinUI/pull/503#discussion_r816922553

Reply:
huh? But dont you want to apply that rule only when vertical is chosen (and not some random stuff) ?

Reply:
by that logic you would never use `@else`

Reply:
I really dont understand , where are you going with this or not having else :DD
But w/e, its minor not worth spending hours..